### DISCUSSION TITLE
how to get the week's start and end date on the highlight weeks

### DISCUSSION THREAD

**Discussion by rickyanwar** [wrick17/calendar-plugin#16] [Q&A]
how to get the week's start and end date on the highlight weeks
![image](https://user-images.githubusercontent.com/36995966/208732248-354740a4-67a8-40ad-a7cb-221932a5c26b.png)
how I get date 15 and 21

**Comment by wrick17** [comment]
> how to get the week's start and end date on the highlight weeks
> 
> ![image](https://user-images.githubusercontent.com/36995966/208732248-354740a4-67a8-40ad-a7cb-221932a5c26b.png)
> 
> how I get date 15 and 21

Don't have a mechanism for that yet. Will update this thread if and when I can implement.
